test/etf_encode_test.py

```diff
@@ -506,7 +506,9 @@ def test_fail_encode_nonfinite_float_native(self):
         self._fail_encode_nonfinite(native_impl)
 
     def _fail_encode_nonfinite(self, codec):
-        nonfinites = [float(n) for n in "inf -inf nan".split()]
+        # Check that nonfinites raise exception, even when within a list, tuple
```

Comment:
for better resolution in where a regression might apear, could we add another tests rather than changing this one